CaptureSDK/GiniCaptureSDK/Documentation/source/Customization guide.md

```diff
@@ -112,105 +118,32 @@ During QR Code only mode the capture and import controls will be hidden from the
-
-##### 2. Main collection
-- Opaque view style (when tool tip is shown)  &#8594;  `GiniConfiguration.multipageToolTipOpaqueBackgroundStyle`
-
-##### 3. Page item
-- Page upload state icon
-  - Successful upload &#8594; <span style="color:#009EDF">*successfullUploadIcon*</span> image asset
-  - Failed upload &#8594; <span style="color:#009EDF">*failureUploadIcon*</span> image asset
-- Page upload state icon background color
-  - Successful upload &#8594; `GiniConfiguration.multipagePageSuccessfullUploadIconBackgroundColor`
-  - Failed upload &#8594; `GiniConfiguration.multipagePageFailureUploadIconBackgroundColor`
-- Page circle indicator color &#8594; `GiniConfiguration.indicatorCircleColor` using `GiniColor` with dark mode and light mode colors
-- Page indicator color &#8594; `GiniConfiguration.multipagePageIndicatorColor` 
-- Page background color &#8594; `GiniConfiguration.multipagePageBackgroundColor` using `GiniColor` with dark mode and light mode colors
-- Page selected indicator color &#8594; `GiniConfiguration.multipagePageSelectedIndicatorColor`
-- Page draggable icon tint color &#8594; `GiniConfiguration.multipageDraggableIconColor`
-
-##### 4. Bottom container
-- Background color &#8594; `GiniConfiguration.multipagePagesContainerAndToolBarColor` using `GiniColor` with dark mode and light mode colors
-- Rotation button image &#8594;  <span style="color:#009EDF">*rotateImageIcon*</span> image asset
-- Delete button image &#8594;  <span style="color:#009EDF">*trashIcon*</span> image asset
+### Camera import error handling TODO
 
-## Analysis screen
+## Review screen TODO
+
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="600" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2F1985HMF83siAXmysSn3dC6%2FiOS-Gini-Capture-SDK-3.0.0-UI-Customisation%3Fnode-id%3D261%253A8256%26t%3DcRAvcUKVlwGtGpuh-1" allowfullscreen></iframe>
 
-<br>
-<center><img src="img/Customization guide/Analysis.jpg" height="500"/></center>
-</br>
+## Analysis screen
 
-##### 1. Navigation bar
-- Cancel button
-  - With image and title
-	  - Image &#8594; <span style="color:#009EDF">*navigationAnalysisBack*</span> image asset
-	  - Title &#8594; <span style="color:#009EDF">*ginicapture.navigationbar.analysis.back*</span> localized string
-  - With title only
-	  - Title &#8594; `GiniConfiguration.navigationBarAnalysisTitleBackButton`
+<iframe style="border: 1px solid rgba(0, 0, 0, 0.1);" width="600" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2F1985HMF83siAXmysSn3dC6%2FiOS-Gini-Capture-SDK-3.0.0-UI-Customisation%3Fnode-id%3D501%253A7494%26t%3DcRAvcUKVlwGtGpuh-1" allowfullscreen></iframe>
 
-##### 2. PDF Information view
-- Text color &#8594; `GiniConfiguration.analysisPDFInformationTextColor`
-- Background color &#8594; `GiniConfiguration.analysisPDFInformationBackgroundColor`
+You can show a custom loading indicator with custom animation support.
```

Comment:
[Suggestion] Maybe we should mention here that the same loading indicator will be also used when a correct QR code is scanned.

Reply:
Added it for QR code scanning 😉